js/messaging.js

```diff
@@ -387,3 +387,16 @@ function invokeOrPostpone(isInvoke, fn, ...args) {
     ? fn(...args)
     : setTimeout(invokeOrPostpone, 0, true, fn, ...args);
 }
+
+
+function openEditor(id) {
+  let url = chrome.runtime.getURL('edit.html');
```

Comment:
chrome.runtime.getURL is not needed when opening internal URL from the code that runs within own page like messaging.js:
```const url = '/edit.html' + (id ? `?id=${id}` : '');```